### PR TITLE
add alertSnackbar component, delete error and warning snackbars

### PR DESCRIPTION
**Pull request by Junk-debug**

### CODE REVIEW

**Comment by Junk-debug** [comment]
@Softjey please review this pull request

File: client/src/components/UI/AlertSnackbar.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+const getDefaultVariant = (severity: AlertProps['severity']) => {
+  switch (severity) {
+    case 'error':
+    case 'warning':
+      return 'filled';
+    default:
+      return undefined;
+  }
+};
+
```
Comment (Softjey) [inline client/src/components/UI/AlertSnackbar.tsx:33]: Is better if you just hardcode this props in component

File: client/src/components/UI/AlertSnackbar.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  alertProps: { sx: alertSx, ...alertProps } = {},
+  ...props
+}) => {
+  const handleClose = (_: React.SyntheticEvent | Event, reason?: string) => {
+    if (reason === 'clickaway') {
+      return;
+    }
+
+    onClose();
+  };
```
Comment (Softjey) [inline client/src/components/UI/AlertSnackbar.tsx:60]: Is will be more readable if you add positive case check instead of negative

Comment (Junk-debug) [reply client/src/components/UI/AlertSnackbar.tsx:60]: here i think my variant is more readable, btw i got it from mui docs

**Review by Softjey — APPROVED** [review]

**Pull request merged**